spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/util/ExportUtil.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2009 - 2022 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.spec.bootstrap.data.util;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+/**
+ * Utility methods for managing manifest exports.
+ */
+public final class ExportUtil {
+    private static final String EXPORT_NAME = "consumer_export.zip";
+
+    private ExportUtil() {
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * Retrieves the export zip file in the manifest.
+     *
+     * @param manifest compressed manifest to retrieve the export from.
+     * @return a compressed {@link ZipFile} of the export.
+     * @throws IllegalArgumentException if manifest is null.
+     * @throws IOException if unable to create a temporary export zip file.
+     */
+    public static ZipFile getExportArchive(File manifest) throws IOException {
+        if (manifest == null) {
+            throw new IllegalArgumentException("Manifest is null.");
+        }
+
+        try (ZipFile zipFile = new ZipFile(manifest)) {
+            ZipEntry entry = zipFile.getEntry(EXPORT_NAME);
+            File tmp = File.createTempFile("export", ".zip");
+            tmp.deleteOnExit();
```

Comment:
I meant to add a comment on this because I was confused on why the `File.createTempFile` would not cleanup the file when the VM terminates. Taking a look at the java docs, it mentions that you have to invoke the deleteOnExit method if you want the file to be deleted automatically.

https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/File.html#createTempFile(java.lang.String,java.lang.String,java.io.File)

Reply:
That's unfortunate. Good catch.